agent/bench-scripts/pbench-run-benchmark

```diff
@@ -208,9 +208,9 @@ $SIG{INT} = signal_handler;
 $SIG{QUIT} = signal_handler;
 $SIG{TERM} = signal_handler;
 
-# start tool meister
-system("pbench-tool-meister-start " . $params{"tool-group"});
-metadata_log_begin_run($base_bench_dir, $benchmark, $params{"tool-group"});
+# start tool meister ensuring benchmark, config, and date ENVs are presented so
+# that they are recorded by the Tool Data Sink in the metadata.log file.
+system("benchmark=" . $benchmark_name . " config=" . $params{"user-tags"} . " date=" . $date . " pbench-tool-meister-start " . $params{"tool-group"});
```

Comment:
Let's consider keeping `metadata_log_begin_run` instead of replacing it with a `system()` call.